src/_common/content/content-viewer/content-viewer.ts

```diff
@@ -22,6 +27,18 @@ export default class AppContentViewer extends Vue implements ContentOwner {
 	data: ContentDocument | null = null;
 	hydrator: ContentHydrator = new ContentHydrator();
 
+	isInview = GJ_IS_SSR;
+	mediaItemsNeedRefreshed = true;
```

Comment:
This is a bit misleading. It's not that it's in view if we're doing SSR, its that if we're doing SSR we need to fetch the media items regardless of whether its in view. I think we should check for GJ_IS_SSR in setContent instead of this. feels hacky.